counterpartylib/lib/config.py

```diff
@@ -7,7 +7,7 @@
 # Versions
 VERSION_MAJOR = 9
 VERSION_MINOR = 55
-VERSION_REVISION = 1
+VERSION_REVISION = 2
```

Comment:
should be `MINOR` bump I think? but we usually bump this in `master` before we do a release, but leave this for now so it triggers a reparse for people trying to run the PR I suppose